test/e2e-v2/cases/elasticsearch/docker-compose.yml

```diff
@@ -0,0 +1,79 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2.1'
+
+services:
+  oap:
+    extends:
+      file: ../../script/docker-compose/base-compose.yml
+      service: oap
+    ports:
+      - 12800
+    networks:
+      - e2e
+  es:
+    image: elastic/elasticsearch:8.3.2
+    expose:
+      - 9200
+    networks:
+      - e2e
+    environment:
+      - discovery.type=single-node
+      - cluster.routing.allocation.disk.threshold_enabled=false
+      - xpack.security.enabled=false
+      - cluster.name=cluster-e2e
+      - node.name=node-e2e
+    healthcheck:
+      test: ["CMD", "bash", "-c", "cat < /dev/null > /dev/tcp/127.0.0.1/9200"]
+      interval: 5s
+      timeout: 60s
+      retries: 120
+  es-exporter:
+    image: quay.io/prometheuscommunity/elasticsearch-exporter:v1.5.0
+    expose:
+      - 9108
+    networks:
+      - e2e
+    environment:
+      - ES_URI=http://es:9200
+    depends_on:
+      - es
+      - oap
+  es-mock:
+    image: curlimages/curl:latest
+    networks:
+      - e2e
+    depends_on:
+      - es
+      - es-exporter
+      - oap
+    command: "sh -c ./es-e2e-data-mock.sh"
```

Comment:
Could you explain why we need mock data here?

Reply:
Push the data into the server to build metrics?

Reply:
Ok,  make sense :)